### ISSUE TITLE
hguemar is displayed very badly

### ISSUE THREAD

**Issue by keszybz** [FrostyX/fedora-sponsors#9]
I see "HaÃ¯kel GuÃ©mar". `/msg zodbot fasinfo hguemar` looks OK… No idea what went wrong with the website.

**Comment by FrostyX** [comment]
Thank you for the report @keszybz,
IDK what's wrong either. If I open `hguemar`'s profile in FAS
https://accounts.fedoraproject.org/user/hguemar/
his name is rendered incorrectly there as well.

I guess FAS returns the name already broken, let me check.

**Comment by keszybz** [comment]
Indeed. So it looks like the problem is elsewhere.

@hguemar: please fix your FAS entry!

**Comment by FrostyX** [comment]
```python
from fasjson_client import Client
client = Client("https://fasjson.fedoraproject.org/")
hguemar = client.get_user(username="hguemar")
print(hguemar.result["human_name"])
```

```
HaÃ¯kel GuÃ©mar
```

I think this is an issue on FAS side. I will submit a ticket for that.

**Comment by FrostyX** [comment]
Reported: https://github.com/fedora-infra/noggin/issues/686

**Comment by FrostyX** [comment]
Fixed by @abompard in the fedora-infra/noggin#686
